portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,6 +37,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String email = request.getParameter("email-input");
         String subject = request.getParameter("subject-input");
         String message = request.getParameter("message-input");
+        long timestamp = System.currentTimeMillis();
 
         // Error handling - don't allow empty or null values
```

Comment:
The comment says "empty or null values", but the code below actually only checks for null values. There is a helper method for isNullOrEmpty() [1]

[1] https://guava.dev/releases/20.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty-java.lang.String-